zokrates_core_test/tests/tests/snark/snark_verify_bls12_377_2.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Would be nice to have a hint on how to generate these tests. Maybe in the zok file.

Reply:
Not sure what to write here, I've included some comments that might help (the underlying circuit, etc)

Reply:
Is it trivial how to go from the vk generated by the setup to the inputs to this function? Same for the proof?

Reply:
wrote some instructions, lmk what you think